Jenkinsfile

```diff
@@ -0,0 +1,46 @@
+pipeline {
+    options {
+        buildDiscarder(logRotator(numToKeepStr: '20'))
+        timestamps()
+    }
+    agent {
+        docker {
+            label 'd3-build-agent'
+            image 'openjdk:8-jdk-alpine'
+            args '-v /var/run/docker.sock:/var/run/docker.sock -v /tmp:/tmp'
```

Comment:
`args` can be removed completely if Docker socket passthrough is not needed (e.g., if you do not run tests in container inside of another container)